common/key-util/src/test/java/io/helidon/common/pki/KeyConfigTest.java

```diff
@@ -155,10 +156,10 @@ void testDoublePath() {
 
     @Test
     void testPem() {
-        KeyConfig conf = KeyConfig.pemBuilder()
-                .certChain(Resource.create("keystore/public_key_cert.pem"))
-                .key(Resource.create("keystore/id_rsa.p8"))
-                .keyPassphrase("heslo".toCharArray())
+        Keys conf = Keys.builder()
```

Comment:
I thought we decided as a convention to require/mandate all config have a suffix named Config.

Reply:
This is not a config, this is the result type. 
We can discuss this in team